frodo.js

```diff
@@ -2,125 +2,131 @@
  * Extras Plugin
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!